build/lib/release.sh

```diff
@@ -386,7 +390,7 @@ EOF
 	if [[ "${arch}" == "linux/s390x" || "${arch}" == "linux/ppc64le" ]]; then
           DOCKER_CLI_EXPERIMENTAL=enabled "${DOCKER[@]}" buildx build --platform linux/"${arch}" --load ${docker_build_opts:+"${docker_build_opts}"} -q -t "${docker_image_tag}" "${docker_build_path}" >/dev/null
 	else
-          "${DOCKER[@]}" build ${docker_build_opts:+"${docker_build_opts}"} -q -t "${docker_image_tag}" "${docker_build_path}" >/dev/null
+          DOCKER_BUILDKIT=1 "${DOCKER[@]}" build ${docker_build_opts:+"${docker_build_opts}"} -q -t "${docker_image_tag}" "${docker_build_path}" --build-arg arch="${arch}" --build-arg base_image="${base_image}" >/dev/null
```

Comment:
DOCKER_BUILDKIT=1 is required so that xattrs are persisted across stages in multi-stage docker builds. See https://github.com/moby/moby/issues/38132.

Reply:
oh interesting, we've actually been using `DOCKER_CLI_EXPERIMENTAL=enabled buildx build` as above.

WDYT about just using the buildx approach above on all arches?

also: we need to be ensuring buildx/qemu is initialized, I'm not sure if we are for this build yet.

Reply:
Yeah building all images using buildx approach makes sense. Updated it to use buildx.

Reply:
Looks like some one beat me to the punch https://github.com/kubernetes/kubernetes/pull/98529